integration-tests/config.ts

```diff
@@ -205,10 +201,12 @@ const faucetKeyFile = process.env['TEZOS_FAUCET_KEY_FILE'];
 const setupForger = (Tezos: TezosToolkit, forger: ForgerType): void => {
   if (forger === ForgerType.LOCAL) {
     Tezos.setProvider({ forger: localForger });
-  } else if (forger === ForgerType.COMPOSITE) {
+  } if (forger === ForgerType.COMPOSITE) {
```

Comment:
I think we should put this in the next line since it's now a different if scope

Reply:
Oops didn't mean to remove it, I put the else back, thanks.